app/src/main/java/grinnell/appdev/edu/lyles/MenuItem.java

```diff
@@ -0,0 +1,73 @@
+package grinnell.appdev.edu.lyles;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+
+/**
+ * A model to describe menu items
+ *
+ * @author Shelby Frazier
+ */
+
+public class MenuItem {
+
+    private static final double DEFAULT_PRICE = 0;
```

Comment:
We should either use the `Constants` class or have constants live in individual classes.